### PR TITLE
Add Sway icon

### PR DESCRIPTION
**Pull request by eeelbrens**
sway

~**Issue:** closes #~

**Popularity metric:**

[Star History Chart](https://star-history.com/#swaywm/sway&Date)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Logo vectorized from [source](https://github.com/swaywm/sway/blob/master/assets/Sway_Tree.svg).
Hex value `#68751C` picked as most predominant in logo (there's no official designated brand color/identity in Sway, only logo versions present); if anything, blue in their assets/'s wallpaper is not chosen to distinguish them from i3 and Hyprland's colors.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -15100,6 +15100,16 @@
             "hex": "FFA633",
             "source": "https://foursquare.com/about/logos"
         },
+        {
+            "title": "Sway",
+            "hex": "68751C",
+            "source": "https://github.com/swaywm/sway/blob/master/assets/Sway_Tree.svg",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:15106]:
```suggestion
            "source": "https://github.com/swaywm/sway/blob/77b9ddabe2a97c5d04c30929b0f8cbde3470fdd7/assets/Sway_Tree.svg",
```
Please change to permalink when linking from GitHub

File: _data/simple-icons.json [hunk 2]
```diff
@@ -15100,6 +15100,16 @@
             "hex": "FFA633",
             "source": "https://foursquare.com/about/logos"
         },
+        {
+            "title": "Sway",
+            "hex": "68751C",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:15105]: @simple-icons/pr-reviewers What do you guys think about the choice of color for this multi-color logo?

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
@eeelbrens please make the requested changes. The icon matches well otherwise. 👍

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for your contribution @eeelbrens 🎉 
Just waiting for consensus on the color

File: icons/sway.svg
Comment (PratyushJayachandran) [inline icons/sway.svg:1]: Please rearrange the attributes like this
`<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">`

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Comment by eeelbrens** [comment]
@PratyushJayachandran okay so how it's going on your end?

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran okay so how it's going on your end?

I'm okay with the color choice that is why I marked the PR as approved. I wanted at least one more reviewer to confirm.

**Comment by uncenter** [comment]
Will take a look tomorrow and hopefully get this through :)

**Pull request merged**